docs/webrtc/nanostream_webrtc_audio_video_only.md

```diff
@@ -0,0 +1,28 @@
+---
+id: nanostream_webrtc_audio_video_only
+title: Audio- / Video-Only
+sidebar_label: Audio- / Video-Only
+---
+
+In addition to regular streaming, with the nanoStream Webcaster you stream Audio-Only and Video-Only:<br>
+
+```javascript
+// to start "Audio-Only" or "Video-Only" streaming, 
+// we pass "false" to the video or audio device configuration
+
+ var videoDeviceConfig = {
```

Comment:
Can we have two snippets, one for audio only, one for video only?
I fear that customers will simply copy this here...